api/v1alpha4/tags_test.go

```diff
@@ -155,7 +156,7 @@ func TestTags_Difference(t *testing.T) {
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
 			out := tc.self.Difference(tc.input)
-			if e, a := tc.expected, out; !reflect.DeepEqual(e, a) {
+			if e, a := tc.expected, out; !cmp.Equal(e, a) {
```

Comment:
I am new to the community and randomly just came across this PR but I think in these cases, `cmp.Diff` would be more appropriate. Instead of the whole output (`("expected %#v, got %#v", e, a)`) only the diffs will be spitted out. Even I would say, all `cmp.Equal` can be replaced with `cmp.Diff`. 
>Diff returns a human-readable report of the differences between two values: y - x. It returns an empty string if and only if Equal returns true for the same input values and options.

ref https://pkg.go.dev/github.com/google/go-cmp/cmp#Diff

Guys, what do you think? Truly sorry for the sudden unexpected review.

Reply:
Thanks for comment @bisakhmondal, we truly appreciate your opinions, so please don't hesitate.
IMO there is not truly requirement to computing diff here, as we are concerned only to compare two interfaces (wherever reflect.DeepEqual was used). So it would be better to use cmp.Equal, but if you have any more pros of using Diff, please feel free to comment.